docs/README.md

```diff
@@ -1,43 +1,92 @@
+| -------------- | ----------- |
+| `screenly-anthias-nginx-1` | NGINX service |
+| `screenly-anthias-viewer-1` | Viewer service |
+| `screenly-anthias-celery-1` | Celery service |
+| `screenly-anthias-websocket-1` | WebSocket service |
+| `screenly-anthias-server-1` | web UI (front-end and back-end) |
+| `screenly-anthias-redis-1` | Redis (database, cache, message broker) |
+| `screenly-anthias-wifi-connect-1` | Wi-Fi connectivity |
+
+
+### Using `docker-compose logs`
+
+Before running the succeeding commands, make sure that you're in the
+`/home/${USER}/screenly` directory:
+
+```bash
+$ cd /home/${USER}/screenly # e.g., /home/pi/screenly if the user is `pi`
+```
+
+If you'd like to see the logs of a specific container or service via Docker Compose,
+you can run the following:
+
+```bash
+$ docker compose logs -f ${SERVICE_NAME}
+# e.g., docker compose logs -f anthias-server
+```
+
+Check out [this section](/docs/developer-documentation.md#understanding-the-components-that-make-up-anthias) of the Developer documentation page for the list of available services.
+
+## Enabling SSH
 
 See [the official documentation](https://www.raspberrypi.org/documentation/remote-access/ssh/)
 
-# Updating Screenly
-From a Console Run:
+## Updating Anthias
+
+Run the following command in your console:
+
+```bash
+$ bash <(curl -sL https://install-anthias.srly.io)
+```
+
+Alternatively, you can also run the following command:
+
+```bash
+$ /home/${USER}/screenly/bin/run_upgrade.sh
+```
+
+## Accessing the REST API
 
-`$ bash <(curl -sL https://www.screenly.io/install-ose.sh)`
+To get started, open your browser and go to `http://<ip-address>:8080/api/docs/`. You should see the Swagger API docs for the endpoints.
 
-# Accessing the SQLite Database
+## Accessing the SQLite Database
 
-**This section is for power users only; DO NOT mess around with the database unless you know what you are doing**.
+**This section is for power users only. **Do not** mess around with the database unless you know what you are doing**.
 
-For most users, it's recommended that you use the API instead.
+For most users, it's recommended that you [use the API instead](#accessing-the-rest-api).
 
 The SQLite Database can be found here: `~/.screenly/screenly.db`
 
-Its a SQLite Database and can be modified with the sqlite3 CLI. The schema is relatively straight forward if you are a developer; the columns of most interest to you will be `name` and `is_enabled`. In addition `start_date` is useful if you want to use this in a disconnected manner.
+Its a SQLite Database and can be modified with the `sqlite3` CLI. The schema is relatively straightforward if you're already familiar. The columns of most interest to you will be `name` and `is_enabled`. In addition, `start_date` is useful if you want to use this in a disconnected manner.
 
-# Wi-Fi Setup
+## Wi-Fi Setup
 
 - Read the [Wi-Fi Setup](wifi-setup.md) page for more details on how to set up Wi-Fi on the Raspberry Pi.
```

Comment:
This Wi-Fi page probably also needs an overhaul.

Reply:
Done. Thanks!
(Changes can be seen [here](https://github.com/Screenly/Anthias/pull/1816/files#diff-4ef6a9c8e5b4796726e15b6a192e7c2fde11bac37841e371c37337d94c4fec12).)